scala/build/Dockerfile.latest

```
@@ -0,0 +1,148 @@
+# Install Yarn
+# Install Node.js and utilities
+# RUN npm install -g npm@latest
+# RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+#     echo "deb https://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list && \
+#     apt-get update && \
+#     apt-get install -y yarn
+# RUN apt-get install -y nodejs npm && \
+#     npm install -g n
+# RUN n stable
+
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
```

Hadolint

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)